python-libraries/nanover-ase/src/nanover/ase/openmm/runner.py

```diff
@@ -188,6 +206,26 @@ def reset(self):
     def step(self):
         self.imd.step()
 
+    def load(self, index: int):
+        self._sim_index = int(index % len(self.simulation_list))
+        self.simulation = self.simulation_list[self._sim_index]
+        self._initialise_calculator(self.simulation, walls=self._imd_params.walls)
+        self._initialise_dynamics()
+        self.imd.replace_dynamics(self.dynamics)
+        self.imd.play()
+
+    def next(self):
+        self.load(self._sim_index + 1)
+
+    def list(self):
```

Comment:
In the rust server I *try* to limit how much I reveal about the server's file system. I manage poorly but telling too much about the structure of the file system may be an issue in a cloud context.

https://github.com/IRL2/nanover-rs/blob/cf86bf1da2ec037dc73b7ec96d169c60ec839ca0/nanover-rs/src/playback.rs#L151

Reply:
Should be fine indeed. It may not be super informative for some users that may call all their files `simulation.xml` but that's more robust than my attempt.